packages/core/src/client.ts

```diff
@@ -0,0 +1,15 @@
+import { GraphQLClient } from 'graphql-request'
+import type { PatchedRequestInit } from 'graphql-request/dist/types'
+import { Exchange } from './exchange'
+import type { Sdk } from './graphql'
+import { getSdk } from './graphql'
+
+export class Client {
+  private readonly sdk: Sdk
```

Comment:
I really have a problem with this `sdk` name. It's really confusing to know what this is referring to.
As this is the last layer before the API, why not naming it simply api?
```suggestion
  private readonly api: Api
```

Reply:
This is not possible with the generator they use sdk as naming and cannot be changed by any config, and I personally don't see the issue. Also, API is not really good as it's a layer to interact with the API and not the API itself.